libraries/loader/src/index.js

```diff
@@ -146,7 +146,7 @@ function loader(componentDirectory, virtualComponents = {}) {
           }
           return new Promise((resolve, reject) => {
             try {
-              resolve(def.setup.call(null, ctx));
+              resolve(def.setup.call(null, ctx, target));
```

Comment:
It'd be great if this was mentioned in the lib-loader README (sorry I didn't say that last review)